src/models/app_data_json_format.rs

```diff
@@ -11,19 +12,32 @@ pub struct Referrer {
     pub version: String,
 }
 
+#[serde_as]
+#[derive(Eq, PartialEq, Clone, Debug, Deserialize, Serialize, Hash, Default)]
+#[serde(rename_all = "camelCase")]
+pub struct Quote {
+    #[serde(with = "u256_decimal")]
```

Comment:
I left out the optional field of the quoteId to not bloat our files unnecessary. We can update it any time later

Reply:
Makes sense to me. We also don't store the quote ID with the order (just the amounts)  in the `orderbook` database. So as of right now, its value is lost anyway.

Reply:
Is the quote ID going to be linked to the order sometime in the near future (or already)? If so, this would be beneficial for other use cases like solver rewards. No need to add it here, just planning ahead.

Reply:
We will need to link quotes to orders for the following two reasons:

1. Match prices to trades for slippage.
2. Evaluate surplus of an order.